sanic/sanic.py

```diff
@@ -314,3 +314,23 @@ def serve_multiple(server_settings, workers, stop_event=None):
             process.terminate()
         for process in processes:
             process.join()
+
+    def log(self, level, msg):
+        '''
+        Logs the message at the specified level.
+        Eg: app.log("INFO","this logs an info")
+        '''
+        from logging import getLogger, basicConfig, INFO
+        basicConfig(
+            level=INFO, format="%(asctime)s: %(levelname)s: %(message)s")
+        log = getLogger(__name__)
+        if level == "DEBUG":
```

Comment:
This can also be done with a dictionary. and maybe be more lenient so that that the level is not case sensitive?